machine_test.go

```diff
@@ -11,6 +11,8 @@
 // express or implied. See the License for the specific language governing
 // permissions and limitations under the License.
 
+//go:generate mockgen -source=machine.go -destination=machine_mock_test.go -package=firecracker
```

Comment:
I think this is going to wipe out the copyright header when `go generate` is run again.  Take a look at https://github.com/aws/amazon-ecs-agent/blob/master/scripts/generate/mockgen.sh for how we've injected the header in the past.

Reply:
Already merged. Will address in next PR.

Reply:
PR to track: https://github.com/golang/mock/pull/234